app/controllers/blogs_controller.rb

```diff
@@ -49,8 +49,8 @@ def set_current_user_blog
   end
 
   def blog_params
-    array_of_permitted_scalars = %i[title content secret]
-    array_of_permitted_scalars.push(:random_eyecatch) if current_user.premium?
-    params.require(:blog).permit(array_of_permitted_scalars)
+    array_of_permitted_filters = %i[title content secret]
```

Comment:
filtersの"s"が配列を表すので、`array_of`は不要ですね。
あと、寿命が短いですし、perimittedされていないfiltersが他にあるわけではないので、単に、

```suggestion
    filters = %i[title content secret]
```

だけでよいと思います。

ちなみに前回コメントしたこれ↓は、「API上の引き数名にあわせて`filters`にすればよいのでは？」とうい意図でした。

<img width="649" alt="Screenshot 2023-06-15 at 8 04 36" src="https://github.com/sochi419/haunted-blog/assets/1148320/d7162fa6-b29a-4b58-8e60-923de8e3fed7">

Reply:
アドバイスありがとうございます🙏
変数名を考えるの難しいですね〜

filtersという命名に修正をしました✍️